src/lazy-loader.marko

```diff
@@ -7,18 +7,24 @@ class {
       componentRenderer: null,
       renderMethod: input.renderMethod || null,
       enhanceModel: input.enhanceModel || null,
+      customLoadEvent: input.customLoadEvent || false,
+      customLoadEventTriggered: false,
```

Comment:
probably dont need a flag for the event being fired. the custom load event will just be called once